public/app/features/alerting/unified/RuleViewer.tsx

```diff
@@ -1,338 +1,30 @@
-                  queryData={data[query.refId]}
-                  relativeTimeRange={query.relativeTimeRange}
-                  evalTimeRange={evaluationTimeRanges[query.refId]}
-                  onEvalTimeRangeChange={(timeRange) => onQueryTimeRangeChange(query.refId, timeRange)}
-                  isAlertCondition={false}
-                />
-              );
-            })}
-          </div>
-        )}
-        {!isFederatedRule && !allDataSourcesAvailable && (
-          <Alert title="Query not available" severity="warning" className={styles.queryWarning}>
-            Cannot display the query preview. Some of the data sources used in the queries are not available.
-          </Alert>
-        )}
-      </Collapse>
-    </RuleViewerLayout>
-  );
-}
-
-function GrafanaRuleUID({ rule }: { rule: GrafanaRuleDefinition }) {
-  const styles = useStyles2(getStyles);
-  const copyUID = () => navigator.clipboard && navigator.clipboard.writeText(rule.uid);
-
-  return (
-    <DetailsField label="Rule UID" childrenWrapperClassName={styles.ruleUid}>
-      {rule.uid} <IconButton name="copy" onClick={copyUID} />
-    </DetailsField>
-  );
-}
-
-function isLoading(data: Record<string, PanelData>): boolean {
-  return !!Object.values(data).find((d) => d.state === LoadingState.Loading);
-}
-
-const getStyles = (theme: GrafanaTheme2) => {
-  return {
-    errorMessage: css`
-      white-space: pre-wrap;
-    `,
-    queries: css`
-      height: 100%;
-      width: 100%;
-    `,
-    collapse: css`
-      margin-top: ${theme.spacing(2)};
-      border-color: ${theme.colors.border.weak};
-      border-radius: ${theme.shape.borderRadius()};
-    `,
-    queriesTitle: css`
-      padding: ${theme.spacing(2, 0.5)};
-      font-size: ${theme.typography.h5.fontSize};
-      font-weight: ${theme.typography.fontWeightBold};
-      font-family: ${theme.typography.h5.fontFamily};
-    `,
-    query: css`
-      border-bottom: 1px solid ${theme.colors.border.medium};
-      padding: ${theme.spacing(2)};
-    `,
-    queryWarning: css`
-      margin: ${theme.spacing(4, 0)};
-    `,
-    title: css`
-      font-size: ${theme.typography.h4.fontSize};
-      font-weight: ${theme.typography.fontWeightBold};
-
-      overflow: hidden;
-      white-space: nowrap;
-      text-overflow: ellipsis;
-    `,
-    details: css`
-      display: flex;
-      flex-direction: row;
-      gap: ${theme.spacing(4)};
-    `,
-    leftSide: css`
-      flex: 1;
-    `,
-    rightSide: css`
-      padding-right: ${theme.spacing(3)};
-
-      max-width: 360px;
-      word-break: break-all;
-      overflow: hidden;
-    `,
-    rightSideDetails: css`
-      & > div:first-child {
-        width: auto;
-      }
-    `,
-    labels: css`
-      justify-content: flex-start;
-    `,
-    ruleUid: css`
-      display: flex;
-      align-items: center;
-      gap: ${theme.spacing(1)};
-    `,
-  };
-};
+import { AlertingPageWrapper } from './components/AlertingPageWrapper';
```

Comment:
So this component now basically just decides which version to render, the contents of the file have been moved to `components/rule-viewer/RuleViewer.v1.tsx` – only the version the user wants is being loaded by using dynamic imports and bundle splitting here.